tests/integration/src/test/java/org/mule/test/components/InvalidFlowStatePlaceholderTestCase.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.test.components;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.internal.matchers.ThrowableMessageMatcher.hasMessage;
+import org.mule.api.MuleContext;
+import org.mule.api.config.ConfigurationException;
+import org.mule.api.lifecycle.InitialisationException;
+import org.mule.config.spring.SpringXmlConfigurationBuilder;
+import org.mule.context.DefaultMuleContextFactory;
+import org.mule.tck.junit4.AbstractMuleTestCase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class InvalidFlowStatePlaceholderTestCase extends AbstractMuleTestCase
```

Comment:
Hey Pablo, I realized that the `AbstractConfigurationErrorTestCase` is under the `mule-module-db` which is not a dependency of the `mule-test-integration`yet. I'm not quite sure if it worths or makes sense to add that dependency for the purpose we're trying to achieve.